PBRVulkan/RayTracer/src/Assets/Shaders/Raytracer/Raytracing.rgen

```diff
@@ -104,11 +104,12 @@ void main()
 	radiance /= float(SPP);
 
 	// HDR scale
-	vec3 accumulatedRadiance = ubo.frame > 1 ? imageLoad(AccumulationImage, ivec2(gl_LaunchIDNV.xy)).rbg : vec3(0);
-	accumulatedRadiance += radiance;	
+	vec4 accumulated = ubo.frame > 1 ? imageLoad(AccumulationImage, ivec2(gl_LaunchIDNV.xy)) : vec4(0.0);
```

Comment:
What was the problem here exactly?

Reply:
Some `GLSL` syntax problem

Reply:
You are not adding the **radiance** anymore.

Reply:
Never mind I saw you add it later!